modules/ols-custom-resource-definition-file-parameters.adoc

```diff
@@ -0,0 +1,498 @@
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  ols:
+    defaultModel:
+|The name of the default model.
+|String
+|Yes
+
+|spec:
+  ols:
+    defaultProvider:
+|Provider that the model uses by default.
+|String
+|No
+
+|spec:
+  ols:
+    maxMemory:
+|Redis maxmemory value.
+|Integer or string. For example, `1024mb`
+|No
+
+|spec:
+  ols:
+    maxMemoryPolicy:
+|Redis `maxmemory` policy. The default value is `allkeys-lru`.
+|String
+|No
+
+|spec:
+  ols:
+    logLevel:
+|The log level. The default is `Info`
+|DEBUG,INFO, WARNING, ERROR, CRITICAL
+|No
+
+|spec:
+  ols:
+    queryFilters:
+|Contains the list of query filters.
+|Array or string.
+|No
+|===
+
+.{ols-short} user data collection CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  ols:
+    userDataCollection:
+|User data collection switches.
+|Object
+|No
+
+|spec:
+  ols:
+    userDataCollection:
+      feedbackDisabled:
+|If you set the parameter to `1` the container does not send user feedback to Red{nbsp}Hat.
+|Boolean
+|No
+
+|spec:
+  ols:
+    userDataCollection:
+      transcriptDisabled:
+|If you set the parameter to `1` the container does not send user transcripts to Red{nbsp}Hat.
+|Boolean
+|No
+|===
+
+.{ols-short} query filter CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  ols:
+    queryFilters:
+      name:
+|Filter name.
+|String
+|No
+
+|spec:
+  ols:
+    queryFilters:
+      name:
+        pattern:
+|Filter pattern.
+|regexp
+|No
+
+|spec:
+  ols:
+    queryFilters:
+      replaceWith:
+|Replacement for matching query. 
+|String
+|No
+|===
+
+.{ols-short} conversation cache CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+   ols:
+     conversationCache:
+       type:
+|Specifies the type of cache used for the AI chat conversation.
+|in_memory, redis, postgres
+|No
+|===
+
+.{ols-short} Redis conversation cache CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+   conversationCache:
+     redis:
+       credentialSecret:
+|Specifies the secret that contains the Redis credentials.
+|String
+|No
+|===
+
+.{ols-short} deployment CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+   ols:
+     deployment:
+       replicas:
+|Specifies the number of {ols-short} pods. The default is 1.
+|Integer
+|No
+|===
+
+.{ols-short} deployment data collector CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+   ols:
+     deployment:
+       dataColector:
+         claims:
+|Lists the resources defined in the `spec.resourceClaims` resource used by this container. To use this field, the `DynamicResourceAllocation` feature must be enabled.
```

Comment:
spec :ols:deployment:dataColector: claims:  - removed this block please

Reply:
removed